.github/workflows/buildAndTest.yml

```diff
@@ -25,14 +25,17 @@ jobs:
           - llvmtype: binary
             llvmbuildtype: out-of-tree
           # Disable M1 builds until https://github.com/llvm/torch-mlir/issues/1094 is fixed
-          - targetarch: AArch64
+          - targetarch: arm64
+            os: ubuntu-22.04
```

Comment:
readability nit: maybe flip this to match the other pair of excludes?
```
- os: ubuntu-22.04
  targetarch: arm64

- os: macos-12
  targetarch: x86_64
```

Reply:
Will address in follow on